modules/arm_plugin/src/transformations/quantize_fusion.cpp

```diff
@@ -179,6 +184,28 @@ ArmPlugin::pass::ConvolutionQuantizeFusion::ConvolutionQuantizeFusion() {
                                                          getFloatVector(fakeQuantize->input_value(2).get_node()),
                                                          getFloatVector(fakeQuantize->input_value(3).get_node()),
                                                          getFloatVector(fakeQuantize->input_value(4).get_node()));
+            if (itMul != pattern_map.end()) {
+                std::vector<float> scales = getFloatVector(pattern_map[q_scale].get_node());
+                if (allEqualToFirst(scales)) {
+                    if (scales[0] == 0.) return false; // Scale multiplier shouldn't be zero
+                    for (int i = 0; i < quantizationInfo.first.size(); ++i) {
```

Comment:
for(auto&& v : quantizationInfo.first)

Reply:
Fixed. I'm still getting used to new c++ style